docs/linux/helper-function/bpf_probe_read_user_str.md

```diff
@@ -13,31 +13,31 @@ description: "This page documents the 'bpf_probe_read_user_str' eBPF helper func
 > Copyright (c) 2015 The Libbpf Authors. All rights reserved.
 
 
-<!-- [HELPER_FUNC_DEF] -->
 Copy a NUL terminated string from an unsafe user address _unsafe_ptr_ to _dst_. The _size_ should include the terminating NUL byte. In case the string length is smaller than _size_, the target is not padded with further NUL bytes. If the string length is larger than _size_, just _size_-1 bytes are copied and the last byte is set to NUL.
```

Comment:
```suggestion
Copy a `NULL` terminated string from an unsafe user address `unsafe_ptr` to `dst`. The `size` should include the terminating `NULL` byte. In case the string length is smaller than `size`, the target is not padded with further `NULL` bytes. If the string length is larger than `size`, just `size-1` bytes are copied and the last byte is set to `NULL`.
```

Oeps, removal of the marker made the spellchecker gang on you. It has an exception for imported/generated text. But since we now "own" this we have to conform to it. Putting references to variables and constants in backticks makes the spellcheck understand that its a code reference and ignores is.